### DISCUSSION TITLE
How to observe Routed Event from parent?

### DISCUSSION THREAD

**Discussion by Snailya** [reactiveui/ReactiveUI#3791] [Q&A]
I have a custom treelist. I want to observe the click event of a button in the item template. Because the item template is changing from list to list, the only way i could came up is to handle the ButtonBase.Click event from this scope of this treelist.
The traditional way to do this is to add a ButtonBase.Click event at the treelist control. However, I wonder if there is a reactiveui way to do so?
`
<controls:TreeListView x:Name="Elements" ButtonBase.Click="Elements_OnClick">`

**Comment by ChrisPulman** [comment]
Hi, you could take a look at this sample, if there is something different that you are trying to do perhaps provide a sample to demonstrate it and I will try to assist. 

https://github.com/ChrisPulman/ReactiveUITreeviewExample

**Reply by Snailya** [reply]
I have created a sample code at https://github.com/Snailya/ListViewSample.

Because the treelistview has a long style, I use a simple ListView instead. In the ListView, the column is customed by setting ListView.View wtih GridViewColumn. One of the Columns is a button. When the user click the button, I want to observe the event and then handle the event in MainWindowViewModel.
